packages/flutter/lib/src/material/spell_check_suggestions_toolbar.dart

```diff
@@ -180,7 +182,7 @@ class SpellCheckSuggestionsToolbar extends StatelessWidget {
     return Padding(
       padding: EdgeInsets.fromLTRB(
         CupertinoTextSelectionToolbar.kToolbarScreenPadding,
-        kToolbarContentDistanceBelow,
+        paddingAbove,
```

Comment:
Is this the right value? I think the menu should be right below the red, misspelled-word-indicating underline, e.g.:

![Screen Shot 2022-09-06 at 3 37 51 PM](https://user-images.githubusercontent.com/43054281/233215239-592e13a4-d784-4452-8cd1-9c8eea38ba83.png)

Reply:
How about this?

<img width="215" alt="Screenshot 2023-04-20 at 12 54 30 PM" src="https://user-images.githubusercontent.com/389558/233474886-ae088057-f0ca-4f31-8541-1d658f9794b5.png">

I think kToolbarContentDistanceBelow doesn't apply to the spell check toolbar then, if we want it right below the anchor.  I removed it from this file, so I'll watch out for a breaking change in case someone was using it already.

Also by the way, the lack of red underline and red selection has me worried.  Is that something we should open an issue for?

Reply:
Ok sounds good! This looks better.

In terms of the red underline/red selection, I revisited some notes I took and seems like that is behavior that differs based on Android version. So, we can open an issue to track it, but just FYI.

Reply:
I created an issue here: https://github.com/flutter/flutter/issues/125424